### ISSUE TITLE
Caldera belongs with LST, not RedHat

### ISSUE THREAD

**Issue by modalpdx** [FabioLolix/LinuxTimeline#83]
Found a problem in the diagram.

Caldera never really had a RedHat-based distro. They were the first company to commercially distribute RedHat in a box set with some proprietary software. The entire suite was titled Caldera Network Desktop, but it wasn’t actually a Linux distro. The installation guide in the box was RedHat-branded and the distro CD was a barely massaged RedHat, essentially raw RedHat with some strategically placed Caldera logos.

I think that got one or two releases, and after that they dropped RedHat and built their own custom distro, OpenLinux, based on LST. OpenLinux lasted all the way through the death of the company. 

Basically, I’d scrap the connection to RedHat and only associate OpenLinux with LST.

**Comment by FabioLolix** [comment]
Hello @modalpdx and thank you for your patience

I don't know I missed that when reading about Caldera/OpenLinux/SCO etc

I had make some work about it 3 months after you posted but eventually lost it

* https://en.wikipedia.org/wiki/Caldera_OpenLinux#Caldera_Network_Desktop

* https://en.wikipedia.org/wiki/Caldera_OpenLinux#Caldera_OpenLinux

**Comment by FabioLolix** [comment]
Fixed https://github.com/FabioLolix/LinuxTimeline/commit/3e1e06e87eecc887fb10b3a6d78554e8f95bb27b
